src/components/dropdown/dropdown.js

```diff
@@ -649,6 +652,15 @@ class Dropdown extends React.Component {
     return results;
   }
 
+  positionOptions() {
```

Comment:
`positionOptions` sounds like it'd return some options - maybe `calculatePosition` or `handlePosition` or something would be more descriptive?

Reply:
done